### ISSUE TITLE
Give the 'back to top' link in the footer an accessible name

### ISSUE THREAD

**Issue by Veyfeyken** [a-ui/core_branding_scss#162]
### Problem

[organisms/organisms.footer](https://a-ui.github.io/core_branding_scss/#organisms-footer)

The 'back to top' icon in the footer is missing an accessible name. The link is empty.

```html
<footer class="o-footer">
  <span class="o-footer__label">© 2017 Stad Antwerpen</span>
  <a href="#" class="o-footer__button a-button a-button--secondary has-icon">
    <span class="fa fa-arrow-up"></span>
  </a>
</footer>
```
![screenshot icon back to top](https://user-images.githubusercontent.com/870049/98689665-6f8ec600-236c-11eb-80c6-87bc1f46d96b.jpg)

### Solution

Add hidden text 'back to top' or an aria-label="back to top".

### Articles and links

[Understanding Success Criterion 1.1.1: Non-text Content](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html)

**Comment by TriangleJuice** [comment]
Fixed in #178
